File: MarketPlace.Backend/MarketPlace/Microservices/Order Service/Services.Order.Domain/Models/Order.cs
namespace OrderService
{
    public class Order
    {
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public List<OrderPoint> OrderPoints { get; set; } = new List<OrderPoint>();
        public double TotalPrice { get; set; }
        public string Status { get; set; } = string.Empty;
        public Guid ControlAdminId { get; set; } 
        public DateTime OrderDateTime { get; set; }
    }
}

File: MarketPlace.Frontend/market-place-client/src/app/data/interfaces/order.ts
export interface Order {
    id: string;
    userId: string;
    orderPoints: OrderPoint[];
    totalPrice: number;
    status: string;     
    controlAdminId: string;           
    orderDateTime: Date;
}

export interface OrderPoint {
    id: string;
    orderId: string;
    productId: string;
    productName?: string | null;    
    productDescription?: string | null;
    productCategory?: string | null;
    productType?: string | null;
    productImage?: string | null;
    numberOfUnits: number;          
}
  